dotnet/src/SemanticKernel/Kernel.cs

```diff
@@ -36,7 +36,11 @@ public sealed class Kernel : IKernel, IDisposable
     public KernelConfig Config { get; }
 
     /// <inheritdoc/>
-    public ILogger Log { get; }
+    [Obsolete("Use Logger instead. This will be removed in a future release.")]
```

Comment:
Consider putting [EditorBrowsable(EditorBrowsableState.Never)] on these obsolete things until we rip them out.

Reply:
OOooo good idea!
Filed #2191 and will make a full pass (another PR).